eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -24,6 +24,48 @@
 TMPDIR = '/tmp/'
 
 
+def output_report(n_total_clinvar_records, evidence_string_list, n_processed_clinvar_records, unrecognised_clin_sigs,
```

Comment:
I think it is not necessary to declare this function before `clinvar_to_evidence_strings`: it won't be interpreted until that function is called and by then it will have been already parsed. Placing it after the main function will make the code flow more obvious.